COPY/usr/bin/manageiq-db-ready

```diff
@@ -36,11 +36,23 @@ def manageiq_db_ready?(db_yaml, env = "production")
   end
 end
 
+def manageiq_db_pending_migrations?
+  db_migrate_status       = `cd /var/www/miq/vmdb && bin/rake db:migrate:status 2>/dev/null`
```

Comment:
I'm not sure why we didn't use AwesomeSpawn above, this isn't run from the gemset context but I think we should be able to use awesome spawn here but I need to double check that.

Reply:
We only really need awesomespawn if user data could get to the execution strings.  Otherwise this is fine and probably better to have one less dep

Reply:
@agrare I think it might be simpler to run `bin/rake db:abort_if_pending_migrations`